src/DependencyInjection/Compiler/KernelPass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\PlatformHttpCacheBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ *
+ */
+class KernelPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        // slots
+        foreach ($container->getDefinitions() as $id => $definition) {
+            if ($this->isSignalSlot($id) ||
+                $this->isSmartCacheListener($id) ||
+                $this->isCacheTagListener($id)
+            ) {
+                $container->removeDefinition($id);
+            }
+        }
+    }
+
+    /**
+     * @param string $id
+     *
+     * @return bool
+     */
+    protected function isSignalSlot($id)
```

Comment:
Went for `isCacheSlot`. I think that "ez" and "http" are implicit given the vendor and package.